sympy/solvers/ode/tests/test_ode.py

```diff
@@ -1521,74 +1521,6 @@ def test_exact_enhancement():
     assert checkodesol(eq, sol, order=1, solve_for_func=False) == [(True, 0), (True, 0)]
 
 
-@slow
-def test_separable_reduced():
-    f = Function('f')
-    x = Symbol('x')
-    df = f(x).diff(x)
-    eq = (x / f(x))*df  + tan(x**2*f(x) / (x**2*f(x) - 1))
-    assert classify_ode(eq) == ('separable_reduced', 'lie_group',
-        'separable_reduced_Integral')
-
-    eq = x* df  + f(x)* (1 / (x**2*f(x) - 1))
-    assert classify_ode(eq) == ('separable_reduced', 'lie_group',
-        'separable_reduced_Integral')
-    sol = dsolve(eq, hint = 'separable_reduced', simplify=False)
-    assert sol.lhs ==  log(x**2*f(x))/3 + log(x**2*f(x) - Rational(3, 2))/6
-    assert sol.rhs == C1 + log(x)
-    assert checkodesol(eq, sol, order=1, solve_for_func=False)[0]
-
-    eq = f(x).diff(x) + (f(x) / (x**4*f(x) - x))
-    assert classify_ode(eq) == ('separable_reduced', 'lie_group',
-        'separable_reduced_Integral')
-    sol = dsolve(eq, hint = 'separable_reduced')
-    # FIXME: This one hangs
-    #assert checkodesol(eq, sol, order=1, solve_for_func=False) == [(True, 0)] * 4
-    assert len(sol) == 4
-
```

Comment:
@oscarbenjamin I wanted suggestion for this. Whole function is marked as slow just because of this example as when we call `dsolve(eq, hint = 'separable_reduced')` it takes around 4-5 minutes whereas when called like `dsolve(eq, hint = 'separable_reduced', simplify=False)` It returns the solution instantly and checkodesol still hangs for this.
So in the refactored examples should I call it with `simplify=False` or same as like this and mark it `slow=True`

Reply:
We could have a test with simplify=False and then another which is marked as too slow.

Reply:
so basically two seperate examples should be kept for this equation?

Reply:
Yes, although we can avoid duplication by creating a separate function for returning the equation.

Reply:
could you please elaborate more about creating separate function like what it should contain or something?

Reply:
Basically we should normally define the equation in one place even if we are going to use it in multiple places. Exactly how we do that doesn't matter too much.